news/read-direct-fix.rst

```diff
@@ -0,0 +1,29 @@
+New features
+------------
+
+* <news item>
+
+Deprecations
+------------
+
+* <news item>
+
+Exposing HDF5 functions
+-----------------------
+
+* <news item>
+
+Bug fixes
+---------
+
+* Fix ``read_direct`` when the source and destination have different shapes.
```

Comment:
```suggestion
* Fix :meth:`Dataset.read_direct` when the source and destination have different shapes.
```

This should ultimately create a nice link when it's combined into the release notes.

Reply:
Should it have a dot before `Dataset` so that Sphinx will search for it? Or is `h5py` already considered the current module for the release notes?

Reply:
I'm not 100% sure, to be honest. There always has to be some manual editing of the release notes when preparing a release, so I'd catch it then, but I don't think there's any harm in putting the dot anyway if you prefer.